site/gatsby-site/playwright/e2e/unit/gtag.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { expect } from '@playwright/test';
+import config from '../../../config';
+import { test } from '../../utils';
+
+const productionUrl = config.gatsby.siteUrl;
+const googleTrackingId = config.gatsby.gaTrackingId;
+
+test.describe('Google Analytics Tracking', () => {
+  test('Should track gtag only on production', async ({ page }) => {
+    await page.goto('/');
+
+    const currentUrl = page.url();
+
+    const isProduction = currentUrl.startsWith(productionUrl);
```

Comment:
Are we sure that the `currentUrl` in production will be `https://incidentdatabase.ai` in the testing environment? If not, this test should be changed.

Reply:
Note: we can confirm that the `currentUrl` during the test run is not `https://incidentdatabase.ai`, it's `localhost`, so this logic should be changed.